Packages/com.unity.inputsystem/Documentation~/Migration.md

```diff
@@ -3,15 +3,25 @@ uid: input-system-migration
 ---
 # Migrating from the old Input Manager
 
+- [Read the introductory documentation first](#read-the-introductory-documentation-first)
+- [Which system is enabled?](#which-system-is-enabled)
+- [List of corresponding API in the old Input Manager new Input System package](#list-of-corresponding-api-in-the-old-input-manager-new-input-system-package)
```

Comment:
Should probably say
in the old Input Manager vs new Input System package

rather than just
in the old Input Manager new Input System package

Reply:
Seems updated